src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -674,59 +652,67 @@ private void pinnedEndpointsChanged(List<String> pinnedEndpoints)
      * associated {@code SctpConnection}
      */
     private void onSelectedEndpointChangedEvent(
-            WebRtcDataStream src,
-            JSONObject jsonObject)
+        WebRtcDataStream src,
+        JSONObject jsonObject)
     {
-        List<String> newSelectedEndpointIDs
-                = readSelectedEndpointID(jsonObject);
+        // Find the new pinned endpoint.
+        String newSelectedEndpointID = (String) jsonObject.get("selectedEndpoint");
 
-        if (logger.isDebugEnabled())
+        Set<String> newSelectedIDList = Collections.EMPTY_SET;
+        if (newSelectedEndpointID != null && !"".equals(newSelectedEndpointID))
         {
-            logger.debug(Logger.Category.STATISTICS,
-                         "selected_message," + getLoggingId()
-                         + " selected=" + newSelectedEndpointIDs);
+            newSelectedIDList = Collections.singleton(newSelectedEndpointID);
         }
 
-        Set<Endpoint> newSelectedEndpoints = new HashSet<>();
+        selectedEndpointsChanged(newSelectedIDList);
```

Comment:
same question here...we only parse a single one and override any existing, so is the list useful?